eq-author/src/App/section/Design/SectionEditor/SectionIntroduction/updateSectionIntro.graphql

```diff
@@ -0,0 +1,13 @@
+mutation updateSectionIntroduction($input: UpdateSectionIntroductionInput!) {
+  updateSectionIntroduction(input: $input) {
+    id
+    introductionTitle
+    introductionContent
+    section {
```

Comment:
I don't think we need to do this here. An update never affects the section does it?